### PR TITLE
REVIEW: Create draft of the XMPP Newsletter July 2021

### PR DESCRIPTION
**Pull request by Echolon**
### Final release is planned for **5th of August 2021**.
---
You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: commteam@muc.xmpp.org

- [You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links :pray: 
- Just paste your prepared text in this thread below or in our XMPP channel (xmpp:commteam@muc.xmpp.org?join)
- [Make a pull request by editing the file in this repository](https://github.com/xsf/xmpp.org/blob/newsletter-july2021/content/posts/newsletter/2021-08-05.md)

:arrow_right: Please do not just drop bare links - rather try to make a short and decent formulated description (**<5 lines**) what is the main outcome of your source. You do not need to write in a hard, formal tone - be passionate why this motivates you or what you work on! 

**Note: If you create a pull request, which is highly appreciated, please @mention me once your PR is created!** :pray: 
XMPP Newsletter Timeline

---

This is our planed timeline for this draft:

Status:
- [x] Collection of news
- [ ] Review
- [ ] Blog post + Tiny Newsletter
- [ ] Social media
- [ ] Translations
- [ ] Archived ([Use pandoc!](https://pandoc.org))

### CODE REVIEW

File: content/posts/newsletter/2021-08-05.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+- The French translation appears at [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
+- The German translation appears at [anoxinon.de](https://anoxinon.de/blog/)
+- The Italian translation appears at [NicFab.it](https://www.nicfab.it/la-newsletter-xmpp-di-marzo-2021-versione-italiana/)
+
+Many thanks to the translators and their work! This is a great help to spread the news! Please join them in their work or start over with another language!
+
+## XSF Announcement
+
+Since this month a new sub-domain is available at xmpp.org: data.xmpp.org. Many thanks to MattJ! The first data project being hosted here for everyones access are the [providers lists](https://data.xmpp.net/providers/v1/providers-A.json) from the [XMPP Providers project](https://invent.kde.org/melvo/xmpp-providers). There are already first client implementation using these - please review the [criteria](https://invent.kde.org/melvo/xmpp-providers/-/blob/master/README.md) and add you service as well via the Gitlab repository! Feedback welcome!
```
Comment (wurstsalat3000) [inline content/posts/newsletter/2021-08-05.md:34]: data.xmpp.**org** is a typo, I guess?

File: content/posts/newsletter/2021-08-05.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+## Software news
+
+### Clients and applications
+
+[Gajim News](https://gajim.org/post/2021-07-28-development-news-july/): Development on the new Gajim version continued in July, bringing many fixes and improvements. Also this month: WebSocket improvements and a new python-nbxmpp release.
+
+Profanity [0.11.0](https://github.com/profanity-im/profanity/releases/tag/0.11.0) is out, bringing 6 months of polishing to 0.10.0. This includes MAM support (still experimental), support for changing the password, abilities in MUCs like voice and registration, OMEMO trust mode, MUC-PMs in public channels, spam reporting, server contact discovery, and much more.
+
+[Jan-Philipp Litza](https://github.com/jplitza) and [mortzu](https://github.com/mortzu) build a XMPP feed integration for the German official warning app NINA: [Find the Github repository here](https://github.com/mortzu/nina_xmpp). One can simply [add the bot and register coordinates of interest](https://github.com/mortzu/nina_xmpp/blob/master/README.md). May it never contact you!
```
Comment (wurstsalat3000) [inline content/posts/newsletter/2021-08-05.md:60]: [...] **built an** XMPP feed integration for [...]

File: content/posts/newsletter/2021-08-05.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+UWPX [v.0.34.0.0](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.34.0.0) has been released. This release mostly focuses on bug fixes for the first beta release of UWPX with an ETA of 01.09.2021 and proper push support even if the app is not running. For this [COM8](https://github.com/COM8) has been working on its C++ [push server](https://github.com/UWPX/UWPX-Push-Server) for the last couple of months and it is finally up and running. Besides that, this release also includes XEP-0085 (Chat State Notifications) improvements with a proper [typing indicator](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.34.0.0) and status messages.
+
+## Servers
+
+ejabberd 21.07 has been [released](https://www.process-one.net/blog/ejabberd-21-07/) with a plethora of fixes and improvements, so be sure to read the changelog if you're using shared groups and MySQL. Big changes have been implemented pertaining to the build system, as ejabberd can now be built using [rebar3](https://rebar3.org/) and [Elixir Mix](https://elixir-lang.org/getting-started/mix-otp/introduction-to-mix.html).
+
+## Libraries
+
+No information reached us this month :-(
```
Comment (wurstsalat3000) [inline content/posts/newsletter/2021-08-05.md:70]: [python-nbxmpp 2.0.3](https://gajim.org/post/2021-07-28-development-news-july/) has been released.

**Review by wurstsalat3000 — COMMENTED** [review]
@Echolon

File: content/posts/newsletter/2021-08-05.md (hunk 1 above)
Comment (Echolon) [reply content/posts/newsletter/2021-08-05.md:34]: Very very very attentive!

File: content/posts/newsletter/2021-08-05.md (hunk 2 above)
Comment (Echolon) [reply content/posts/newsletter/2021-08-05.md:60]: I always wonder: "an" would be so much nicer - but indeed, why is this allowed?

**Comment by SamWhited** [comment]
Made a quick pull if it's still possible to get it in since it's something that won't make sense for next month: https://github.com/xsf/xmpp.org/pull/946

**Comment by Echolon** [comment]
:hourglass:  **Editorial closing of the XMPP Newsletter June release**
:rocket: Merge in a few minutes! Docker build pulls it asap :robot:
:pray: Thanks to Adrien Bourmault, Benoît Sibaud, DebXwoody, COM8, emus, mattJ, neox, Licaon_Kter, pmaziere, raspbeguy, wurstsalat3000, Ysabeau for their help in creation, review and translation!
:handshake: ... ehh.. here we go again, https://timetoherd.com/ ... :hand: :mask: :hand:
:heart: :fireworks: **Next release on 5th of August 2021**

**Pull request merged**